Makefile

```diff
@@ -47,8 +47,7 @@ setup/service_account:
 	@oc replace --force -f deploy/role.yaml -n $(NAMESPACE)
 	@oc replace --force -f deploy/service_account.yaml -n $(NAMESPACE)
 	@oc replace --force -f deploy/role_binding.yaml -n $(NAMESPACE)
-	@oc replace --force -f deploy/clusterrole.yaml
-	@cat deploy/cluster_role_binding.yaml | sed "s/namespace: integreatly/namespace: $(NAMESPACE)/g" | oc replace --force -f -
+	@cat deploy/role_binding.yaml | sed "s/namespace: integreatly/namespace: $(NAMESPACE)/g" | oc replace --force -f -
```

Comment:
Updating Makefile as everything consolidated in `role.yaml` & `role_binding.yaml` now